recipes/piqp/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "piqp" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/PREDICT-EPFL/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 9af1fef8308782bd7df2d7aef551cba60d42e983f485e258c9ed23a523cded0c
+
+build:
+  skip: true  # [win and vc<14]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - cython                                 # [build_platform != target_platform]
```

Comment:
I don't see `cython` used in the project. You can remove that line.

Otherwise, looks good!